common/streamdebug.h

```diff
@@ -23,6 +23,10 @@
 #ifndef COMMON_STREAMDEBUG_H
 #define COMMON_STREAMDEBUG_H
 
+#ifdef __MORPHOS__
```

Comment:
Debug is on Common namespace, what is a warning here?

Reply:
Debug, i have this.. i think it's specific MorphOS
![image](https://user-images.githubusercontent.com/36823759/96285972-f8ba1380-0fdf-11eb-85c6-a6b5ff93a60d.png)